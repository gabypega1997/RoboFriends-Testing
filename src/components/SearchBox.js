import React from 'react';
import CounterButton from './CounterButton';

const SearchBox = ({ searchfield, searchChange }) => {
  return (
    <div className='pa2'>
      <div className='ma3 pa2'>
        <CounterButton className="tc"></CounterButton>
      </div>
      <input
        aria-label='Search Robots'
        className='pa3 ba b--green bg-lightest-blue'
        type='search'
        placeholder='search robots'
        onChange={searchChange}
      />
    </div>
  );
}

export default SearchBox;