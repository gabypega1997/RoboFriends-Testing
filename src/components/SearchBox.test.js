import React from "react";
import { shallow } from "enzyme/build";
import SearchBox from "./SearchBox";

let wrapper;
describe("Render SearchBox Component", () => {
  const mockFunktions = {
    searchfield: jest.fn(),
    searchChange: jest.fn(),
  };
  wrapper = shallow(<SearchBox {...mockFunktions} />);

  // it('SearchBox schnapshot',()=>{
  //     expect(wrapper).toMatchSnapshot();
  // })
});
