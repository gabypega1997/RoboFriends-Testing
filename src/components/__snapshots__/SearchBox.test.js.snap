// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SearchBox Component SearchBox schnapshot 1`] = `
<div
  className="pa2"
>
  <div
    className="ma3 pa2"
  >
    <CounterButton
      className="tc"
    />
  </div>
  <input
    aria-label="Search Robots"
    className="pa3 ba b--green bg-lightest-blue"
    onChange={[MockFunction]}
    placeholder="search robots"
    type="search"
  />
</div>
`;
